#! /bin/sh /usr/share/dpatch/dpatch-run
## static_sstrerror.dpatch by Sebastian Harl <sh@tokkee.org>
##
## DP: src/liboping.c: Made sstrerror() static.
## DP:
## DP: This is a private helper function only.

@DPATCH@

diff a/src/liboping.c b/src/liboping.c
--- a/src/liboping.c
+++ b/src/liboping.c
@@ -145,7 +145,7 @@ struct pingobj
 /* Even though Posix requires "strerror_r" to return an "int",
  * some systems (e.g. the GNU libc) return a "char *" _and_
  * ignore the second argument ... -tokkee */
-char *sstrerror (int errnum, char *buf, size_t buflen)
+static char *sstrerror (int errnum, char *buf, size_t buflen)
 {
 	buf[0] = 0;
 
